backend/sky_lectures/urls.py

```diff
@@ -15,7 +15,25 @@
 """
 from django.contrib import admin
 from django.urls import path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+import sky_lectures.settings.base
 
-urlpatterns = [
-    path('admin/', admin.site.urls),
-]
+schema_view = get_schema_view(
+   openapi.Info(
+      title="sky-lectures",
+      default_version='v1',
+      description="Application keeping Sky Gate presentations",
+   ),
+   public=True,
+)
+
+if (sky_lectures.settings.base.DEBUG == True):
+    urlpatterns = [
+        path('admin/', admin.site.urls),
+        path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui')
+    ]
+else:
+    urlpatterns = [
+        path('admin/', admin.site.urls),
```

Comment:
No need to wrap simple if statement with parentheses. Also, equality check with `True` is redundant you could go with `if sky_lectures.settings.base.DEBUG`
Moreover, `else` statement is unnecessary, and the admin pattern is duplicated

```suggestion
urlpatterns = [
    path("admin/", admin.site.urls),
]

if sky_lectures.settings.base.DEBUG:
    urlpatterns += [
        path(
            "swagger/",
            schema_view.with_ui("swagger", cache_timeout=0),
            name="schema-swagger-ui",
        )
    ]
```